tests/lightning_qubit/test_measurements_class.py

```diff
@@ -528,6 +555,17 @@ def test_double_return_value(self, shots, measurement, obs0_, obs1_, lightning_s
                 f"Observable of type {type(obs0_).__name__} is not supported for rotating probabilities."
             )
 
+        rtol = 1.0e-2  # 1% of expected value as tolerance
+        if shots != None and measurement is qml.expval:
+            # Increase the number of shots
+            if isinstance(shots, int):
+                shots *= 10
+            else:
+                shots = [i * 10 for i in shots]
+
+            # Extra tolerance
+            rtol = 5.0e-2  # 5% of expected value as tolerance
+
```

Comment:
Similar case as the above test @multiphaseCFD @AmintorDusko @mlxd

Reply:
Ok. Thanks.